postman/CodeBuddies.postman_collection.json

```diff
@@ -192,26 +189,42 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Search",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{api_url}}/api/v1/resources/?search=benefit",
+							"host": [
+								"{{api_url}}"
+							],
+							"path": [
+								"api",
+								"v1",
+								"resources",
+								""
+							],
+							"query": [
+								{
+									"key": "search",
+									"value": "benefit"
+								}
+							]
+						}
+					},
+					"response": []
 				}
 			],
 			"protocolProfileBehavior": {}
 		}
 	],
-	"auth": {
```

Comment:
Thanks for this, @watchtheblur! I tested by importing your JSON into Postman and trying it out.

I noticed that this blob disappeared: 

```
	"auth": {
		"type": "bearer",
		"bearer": [
			{
				"key": "token",
				"value": "{{token}}",
				"type": "string"
			}
		]
	},
```

... and we'll need it; otherwise I have to add the following manually afterwards:
![image](https://user-images.githubusercontent.com/4512699/90722991-bb9c0400-e270-11ea-8d0f-c16cfc6ab2aa.png)

That's the only issue I saw! The new search endpoint looks good. :) 

I'm tempted to ask you to fix another issue by adding the field `media_type: "BOOKS"` to the "create resources" endpoint, but we can also leave that for another PR.

Reply:
re-imported postman file for the bearer token, and added the search back for exporting.  It looks ok now?  Let me know if something still looks wrong?

Reply:
Thank you! Will review next week!